File: SistemaDecomisos/src/java/model/Distrito.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import java.io.Serializable;

/**
 *
 * @author Marco
 */
public class Distrito implements Serializable, Jsonable{
    private int idDistrito;
    private String nombreDistrito;

    public Distrito() {
    }

    public Distrito(int idDistrito, String nombreDistrito) {
        this.idDistrito = idDistrito;
        this.nombreDistrito = nombreDistrito;
    }

    public int getIdDistrito() {
        return idDistrito;
    }

    public void setIdDistrito(int idDistrito) {
        this.idDistrito = idDistrito;
    }

    public String getNombreDistrito() {
        return nombreDistrito;
    }

    public void setNombreDistrito(String nombreDistrito) {
        this.nombreDistrito = nombreDistrito;
    }
    
    
}

File: web/js/objects/Distrito.js
function Distrito(idDistrito, nombreDistrito) {
    this.Distrito(idDistrito, nombreDistrito);
}

Distrito.prototype = {
    idDistrito: 0, 
    nombreDistrito: "", 
    Distrito: function (idDistrito, nombreDistrito) {
        this.idDistrito = idDistrito, 
        this.nombreDistrito = nombreDistrito;
    }
};

Distrito.from = function (plain) {
    distrito = new Distrito(
    plain.idDistrito, 
    plain.nombreDistrito);
    return distrito;
};

Distrito.to = function (distrito) {
    return {
        _class: 'Distrito',
        idDistrito: distrito.idDistrito, 
        nombreDistrito : distrito.nombreDistrito
    };
};
 